README.md

```diff
@@ -1,21 +1,57 @@
 # perl-Data-Resample
 
-Data::Resample
-
 A module that allows you to resample a data feed
 
-sampling_frequency=15
-tick_cache_size = 30*60
-resample_cache_size = 12*60*60
-redis=xxxx
+#### SYNOPSIS
+
+```
+  use Data::Resample::TicksCache;
+  use Data::Resample::ResampleCache;
+
+  my $ticks_cache = Data::Resample::TicksCache->new({
+        redis => $redis,
+        });
+
+  my %tick = (
+        symbol => 'USDJPY',
+        epoch  => time,
+        quote  => 103.0,
+        bid    => 103.0,
+        ask    => 103.0,
+  );
+
+  $ticks_cache->tick_cache_insert(\%tick);
+
+  my $ticks = $ticks_cache->tick_cache_get_num_ticks({
+        symbol => 'USDJPY',
+        });
+
+  my $resample_cache = Data::Resample::ResampleCache->new({
```

Comment:
I don't understand anything of this example. I am resampling ticks - surely the example should show an array of ticks, and then resample it with a given resampling rate